PFR/WeakPFR.lean

```diff
@@ -771,9 +773,9 @@ lemma single {Ω : Type u} [MeasurableSpace Ω] [DiscreteMeasurableSpace Ω] (μ
   contrapose! hz
   have : Disjoint {z} A := by simp [hz]
   replace this := measureReal_union (μ := μ) this (measurableSet_discrete _)
-  simp [hA] at this
+  simp only [singleton_union, hA] at this
   have h := measureReal_mono (μ := μ) (show insert z A ⊆ Set.univ by simp)
-  simp [this] at h
+  simp only [this, IsProbabilityMeasure.measureReal_univ, add_le_iff_nonpos_left] at h
   assumption
```

Comment:
Maybe this all can be `simpa [this] using h`? Not sure, you should try

Reply:
Yes, it works. Thanks.